### ISSUE TITLE
DJI Air 2S Live Camera Preview ?

### ISSUE THREAD

**Issue by dineshkandan** [dji-sdk/Mobile-SDK-Android#1218]
I have Connected My DJI Air 2S Drone to SDK V4.16.4 via RC .Tried Video Feeder View and XT2 Camera View to Fetch Live Drone Video Preview both are too slow in Updating Frame . Any other way to View Get Live Video Preview of Drone . Kindly Help me to get Live Drone Camera View .Thanks

**Comment by DJI-William** [comment]
What is your Android device model? Some models which have their own chips may have this issue suach as Samsung and Google Pixel.

**Comment by dineshkandan** [comment]
i have tried in pixel 6a and oneplus6t both faces same issue. Any solution for that or any alternative. but Dji Fly App Works without any issue

**Comment by DJI-William** [comment]
Yes, Pixel 6A is a known device for this issue already. This is a MSDK issue and it is coming to fix in 4.16.5.

**Comment by dineshkandan** [comment]
How long it will take to release 4.16.5?

**Comment by Scanflow-admin** [comment]
https://github.com/dji-sdk/Mobile-SDK-Android/issues/1187#issuecomment-1269399625

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/79709'>Zendesk ticket #79709</a>:<br><br>I'm sorry for the long wait. The issue with MSDK 4.X not being able to render videos on some models has been reported and the fix code has been submitted. It will be released with the official version 4.17.
<br><p><em><small>°°°</small></em></p>
